portal-web/docroot/html/taglib/aui/input/init-ext.jspf

```diff
@@ -62,7 +62,17 @@ if (type.equals("toggle-card")) {
 }
 
 if (type.equals("toggle-switch")) {
-	fieldCssClass += " toggle-switch";
+	fieldCssClass += " toggle-switch-check";
+
+	if (Validator.isNotNull(labelCssClass)) {
+		labelCssClass += " ";
+	}
+
+	labelCssClass += "toggle-switch";
```

Comment:
`labelCssClass` should never be `null`. It gets the value `StringPool.BLANK` if the attribute isn't declared.

Reply:
If that's the case, then we don't need
```
if (Validator.isNotNull(labelCssClass)) {
	labelCssClass += " ";
}
```
We can just have
```
labelCssClass += " toggle-switch";
```

Reply:
> Does Java break with NPE on null + "toggle-switch"?
This confused me. Yes `labelCssClass` can be `null` in this file. Yes `null + toggle-switch` will cause NPE, but it will not be `null` because we are adding `StringPool.SPACE` if it is `null` on L68.

It's being done this way so we have the option of passing CSS classes to the `label` element, but not required. The only thing required for toggle switches is the class `toggle-switch` on the `label` element.

Reply:
> it will not be `null` because we are adding `StringPool.SPACE` if it is `null` on L68.

We are adding space if it is **not** `null` on L68
```
if (Validator.isNotNull(labelCssClass)) {
	labelCssClass += " ";
}
```
If it is `null`, it will skip that condition and NPE break on 
```
labelCssClass += "toggle-switch";
```

Reply:
@markocikos yeah you're right, I realized I put the space there because I didn't like the output `class=" toggle-switch"` if there was no `labelCssClass`